src/PairSolver.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { IStrategy, Pool } from "src/interfaces/IStrategy.sol";
+import {
+    computeDeltaXGivenDeltaY,
+    computeDeltaLGivenDeltaX,
+    computeDeltaLGivenDeltaY,
+    computeDeltaYGivenDeltaX,
+    computeDeltaXGivenDeltaL,
+    computeDeltaYGivenDeltaL
+} from "src/lib/StrategyLib.sol";
+
+/**
+ * @title Pair strategy base contract for DFMM.
+ * @notice This abstract contract defines the basic behavior of
+ * a two-token strategy for DFMM. It is meant to be inherited by
+ * a concrete strategy implementation.
+ * @author Primitive
+ */
```

Comment:
wrong natspec

Reply:
Do you want to change it? Not sure what the "right" natspec is. I don't really have any opinions on solidity but all the kits work uses this. I'm more of the opinion on RERO then high modernism.

Reply:
I guess it's time to finish our style guide 👀